app/assets/stylesheets/application.scss

```diff
@@ -1,11 +1,272 @@
 @import "colors";
+@import "foundation_and_overrides";
 @import url(https://fonts.googleapis.com/css?family=Mako);
 @import url(https://fonts.googleapis.com/css?family=Lora);
 
-$font-stack: "Maco", sans-serif;
+$font-stack: 'Mako', sans-serif;
+
 
 h1, h2, h3, h4, h5 {
   font-family: 'Lora', serif;
   margin-bottom: 0rem;
   margin-top: 0.5rem;
 }
+
+body {
+
+}
+
+.top-bar-section {
+    display: inline;
+    height: auto !important;
+    width: auto;
+}
+
+.top-bar-section .dropdown li:not(.has-form):not(.active) > a:not(.button) {
+  background: #333333;
+}
+
+.top-bar-section li.active:not(.has-form) a:not(.button) {
+    min-width: 140px;
+    height: auto;
+    background: #333333;
+}
+
+
+.top-bar-section li:not(.has-form) a:not(.button) {
+    background: #333333;
+}
+
+.top-bar-section li {
+  text-align: center;
+}
+
+.notice {
+  background-color: $fr-green;
+  text-align: center;
+  color: $sc-white;
+}
+
+.alert {
+  background-color: $focus-red;
+  text-align: center;
+  color: $sc-white;
+}
+
+.mbta {
+  color: $sc-white;
+  padding: 0px;
+  margin: 0px;
+  line-height: auto;
+  font-size: 14px;
+  background-color: $focus-red;
+  padding-left: 40px;
+  margin-left: 90px;
+  margin-right: 90px;
+}
+.mbta-header {
+  color: $focus-red;
+  padding-left: 120px;
+  font-weight: bolder;
+}
+
+.logo {
+  padding-left: 30px;
+}
+
+a {
+  color: $white;
+}
+
+a:hover, a:focus {
+  color: $dark-green;
+}
+
+.flash-alert {
+  margin-top: 10px;
+}
+
+.trip-table {
+  margin-left: 200px;
+  margin-right: 200px;
+  background-color: $fr-green;
+}
+
+.trip-table a {
+  color: $fr-black;
+}
+// .review-form-list {
+//   margin: 150px;
+//   padding: 150px;
+// }
+
+.forgot-pass {
+  color: $light-green;
+}
+
+.forgot-pass a {
+  color: $light-green;
+}
+
+.review-title {
+  margin-bottom: 0rem;
+  padding: none;
+}
+
+.park-index-list-item {
+  background-color: $fr-green;
+  list-style-type: none;
+  margin-bottom: 5px;
+  padding: 10px 15px;
+}
+
+.index-details {
+  color: $steel;
+  float: left;
+  font-size: 100%;
+  line-height: 2;
+}
+
+.index-rating p {
+  color: $steel;
+  float: center;
+  font-size: 300%;
+  line-height: 0.6;
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.park-info p {
+  position: relative;
```

Comment:
Properties should be ordered border-color, border-right, position, top, transform